1. Создать форму наследующуюся от `forms.ModelForm`
2. Определить метод clean()
3. Получить `clean_data` => `cleaned_data = super().clean()`
4. Получить нужное поле `archive_role_operator = cleaned_data.get("archive_role_operator", None)`
5. Выдать ValidationError:
	1. Ошибка вылезет при сохранении в начале страницы `from django.core.exceptions import ValidationError`, `ValidationError(f"Ваша организация находится в архиве. Вы не можете получить данную роль",code="archive",)`
	2. Ошибка вылезет только у того поля, в котором допущена ошибка `self.add_error("archive_role_operator", error)`

```python
class PermissionArchiveUserForm(forms.ModelForm):
"""Если организация для ролей Оператор/Администратор ЛКО в архиве, произойдет валидация полей"""

	class Meta:
	models = User

	def clean(self):
		cleaned_data = super().clean()
		error = ValidationError(
		f"Ваша организация находится в архиве. Вы не можете получить данную роль",
		code="archive",
		)
		archive_role_operator = cleaned_data.get("archive_role_operator", None)
		archive_role_admin_lko = cleaned_data.get("archive_role_admin_lko", None)
		user_instance = self.instance
		
		operator_status_department = (
		user_instance.operator_permissions.department.status
		)
		if (
		operator_status_department == DepartmentStatus.ARCHIVED
		and not archive_role_operator
		):
			self.add_error("archive_role_operator", error)
		
		admin_lko_status_department = (
		user_instance.admin_lko_permissions.department.status
		)
		if (
		admin_lko_status_department == DepartmentStatus.ARCHIVED
		and not archive_role_admin_lko
		):
			self.add_error("archive_role_admin_lko", error)
```